3rdparty/README.md

```diff
@@ -0,0 +1,6 @@
+# 3rdparty modules documentation
+
+## AdjustSdk.framework
+Source: https://github.com/adjust/ios_sdk/releases/download/v4.29.4/AdjustSdkDynamic.framework.zip
```

Comment:
Interesting... What about if we write a script to download this package at build-time?
It can be part of the xcode_patcher.rb too. What do you think?